threads/models.py

```
@@ -16,6 +16,7 @@ class Reactions(models.TextChoices):
    SAD = "sad", "Sad"
    ANGRY = "angry", "Angry"
    LAUGH = "laugh", "Laugh"
+   NONE = "none", "None"
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4

(E111)